CPAC/pipeline/engine.py

```diff
@@ -230,26 +244,14 @@ def get(self, resource, pipe_idx=None, report_fetched=False,
                     if report_fetched:
                         return (None, None)
                     return None
-                lookup_message = ("\n[!] C-PAC says: None of the listed "
-                                  "resources are in the resource pool:\n"
-                                  f"{resource}\n")
-                verbose_logger.debug(lookup_message)
-                raise Exception(lookup_message)
+                raise Exception(info_msg)
         else:
             if resource not in self.rpool.keys():
                 if optional:
                     if report_fetched:
                         return (None, None)
                     return None
-                lookup_message = ("\n\n[!] C-PAC says: The listed resource is "
-                                  f"not in the resource pool:\n{resource}\n\n"
-                                  "Developer Note: This may be due to a mis"
-                                  "match between the node block's docstring "
-                                  "'input' field and a strat_pool.get_data() "
-                                  "call within the block function.\n(keys in "
-                                  f"resource pool are {self.rpool.keys()})\n")
```

Comment:
Whoops! Changing this to https://github.com/FCP-INDI/C-PAC/blob/75bbc1bef9e42760ea4a83ec05d72b92df3379c2/CPAC/pipeline/engine.py#L221-L232 

[Breaks](https://app.circleci.com/pipelines/github/FCP-INDI/C-PAC/2718/workflows/ccc9e603-bb6d-4b2c-aa28-96f6e9610407/jobs/14123) this test https://github.com/FCP-INDI/C-PAC/blob/75bbc1bef9e42760ea4a83ec05d72b92df3379c2/CPAC/pipeline/engine.py#L2015-L2019

I think we just need to change https://github.com/FCP-INDI/C-PAC/blob/75bbc1bef9e42760ea4a83ec05d72b92df3379c2/CPAC/pipeline/engine.py#L2019 to 
```Python
    [!] C-PAC says: None of the listed resources are in the resource pool:
```

to get it working again